packages/nimble-components/src/select/index.ts

```diff
@@ -948,7 +1042,7 @@ export class Select
 
     private clearSelection(): void {
```

Comment:
A few thoughts on this `clearSelection` function:
- The name doesn't seem appropriate now that it doesn't have to do with selection
- It seems odd to be clearing the activeOption value on each list option without updating `this.openActiveIndex` back to -1

Is this function even needed with the changes you made to `inputHandler()` so that it always calls `setActiveOption`? I believe that is the only caller of this function.

Reply:
You're right, but this required a slight logic reorganization in `setActiveIndex` so that we _always_ update the `activeOption` state of the previous active index to `false`. (if different than the new one). It was weird, regardless, to have been doing that only when the new active index was valid.